internal/telemetry/tracker.go

```diff
@@ -69,11 +70,18 @@ func newTracker(ctx context.Context, cmd *cobra.Command, args []string) (*tracke
 		storeSet:         storeSet,
 		cmd:              cmd,
 		args:             args,
+		packageSource:    readPackageSource(),
 	}, nil
 }
 
 func (t *tracker) defaultCommandOptions() []eventOpt {
-	return []eventOpt{withCommandPath(t.cmd), withHelpCommand(t.cmd, t.args), withFlags(t.cmd), withProfile(), withVersion(), withOS(), withAuthMethod(), withService(), withProjectID(t.cmd), withOrgID(t.cmd), withTerminal(), withInstaller(t.fs)}
+	ret := []eventOpt{withCommandPath(t.cmd), withHelpCommand(t.cmd, t.args), withFlags(t.cmd), withProfile(), withVersion(), withOS(), withAuthMethod(), withService(), withProjectID(t.cmd), withOrgID(t.cmd), withTerminal(), withInstaller(t.fs)}
+
+	if t.packageSource != nil {
+		ret = append(ret, withPackageSource(*t.packageSource))
+	}
```

Comment:
what's the difference of `withInstaller` and `withPackageSource`? reading the implementation it seems `withInstaller` only supported homebrew, why not expand `withInstaller` with the new checkers, or remove hombrew from `packageSource` since they seem to be overlapping

Reply:
Great catch, I'll merge them